lib/friendly_id/slug_generator.rb

```diff
@@ -12,12 +12,27 @@ def available?(slug)
         return false if @config.reserved_words.include?(slug)
       end
 
+      if @config.treat_numeric_as_conflict && purely_numeric_slug?(slug)
```

Comment:
@oehlschl I just had one last question: if this is only available when using `slugged` (as your docs indicate) then should we check here for `if @config.uses?(::FriendlyId::Slugged)` as well?

Reply:
In other words, it didn't need seem necessary to add since this whole class depends on `Slugged`. But let me know if that's not the case.

Reply:
oh, of course! Thank you